src/tracing/internal/tracing_muxer_impl.cc

```diff
@@ -161,6 +161,18 @@ struct CompareBackendByType {
   }
 };
 
+BufferExhaustedPolicy ParseBufferExhaustedPolicy(const DataSourceConfig& cfg) {
+  switch (cfg.buffer_exhausted_policy()) {
+    case DataSourceConfig::BUFFER_EXHAUSTED_DROP:
+      return BufferExhaustedPolicy::kDrop;
+    case DataSourceConfig::BUFFER_EXHAUSTED_STALL:
+      return BufferExhaustedPolicy::kStall;
+    case DataSourceConfig::BUFFER_EXHAUSTED_STALL_THEN_DROP:
+      return BufferExhaustedPolicy::kStallThenDrop;
+  }
```

Comment:
Here the fact that you don't handle CONFIGURABLE seems a  bug, rather than being deliberate.
IMHO a dedicate boolean is clearer

Reply:
Fair. I'm using a separate boolean now